requirements_dev.txt

```
@@ -1 +1,3 @@
-black>=21.9b0
+black>=21.11b0
+isort>=5.10.0
+pre-commit>=2.15.0
```

```suggestion
black==21.11b1
isort==5.10.1
pre-commit~=2.15.0
```
`requirements_dev.txt` technically doesn't need to list the `black` and `isort`, as `pre-commit` installs its own environments and doesn't use the virtualenv, but I suppose we can keep them in there if someone wants to run one of the tools manually